src/browser/NativeMessageInstaller.cpp

```diff
@@ -58,6 +59,8 @@ namespace
         QStringLiteral("HKEY_CURRENT_USER\\Software\\Chromium\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
     const QString TARGET_DIR_FIREFOX =
         QStringLiteral("HKEY_CURRENT_USER\\Software\\Mozilla\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
+    const QString TARGET_DIR_LIBREWOLF =
+        QStringLiteral("HKEY_CURRENT_USER\\Software\\LibreWolf\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
```

Comment:
Should just set this equal to the FIREFOX variable.

Reply:
Please elaborate?

Reply:
I don't see the technical reason for them to be the same, but I changed it back. Will that be everything required to merge?